Add SC nomination workshop event (#965)
USRSE/usrse.github.io: _events/2022/2022-11-steering-committee-nomination-workshop.md

---
title: Steering Committee Nomination Workshop
location: Virtual 
url: https://asu.zoom.us/j/85894518643?pwd=S0R4MVY1SktUeWJIazYvRkRDUUtlUT09 
expires: 2022-11-03
event_date: "November 2, 2022"
layout: event
category: workshop
time:
    - - start: 2022-11-02T20:00:00Z
        end: 2022-11-02T21:30:00Z
---

> Are you interested in running for steering committee? Come to this informal workshop where we will...

Reviewer: ```suggestion
Are you interested in running for steering committee? Come to this informal informative workshop where we will...
```

> - explain the nomination process and the responsibilities that come with the role

Reviewer: ```suggestion
- discuss the responsibilities that come with the role

- explain the nomination process
```

> - help you generate ideas for the nomination prompts and combat imposters syndrome, especially in the case that you have a unique background

Reviewer: ```suggestion
- help you generate ideas for the nomination prompts and combat imposter syndrome, especially in the case that you have a unique background
```

> - give you enough time and space to complete the entire nomination and ask any questions that come up throughout

Reviewer: ```suggestion
- provide time and space to complete the nomination form and ask any questions that come up throughout
```

> This workshop will take place on [Zoom](https://asu.zoom.us/j/85894518643?pwd=S0R4MVY1SktUeWJIazYvRkRDUUtlUT09). Current committee members will start with a brief presentation on the nomination process and the responsibilities of committee members. We will then work through [a worksheet](https://docs.google.com/document/d/1UB5BVBh9uaZ1thVPeweP2kbEDAZdKWM9l_qBghEbREk/edit?usp=sharing) to help participants generate ideas and material for the nomination statement. Finally, we will give you time to draft your full statement while remaining availble to answer any remaining question you have or provide feedback. We especially encourage you to come to the workshop if you have questions about what constitutes a relevant experience or if you are struggling with imposter syndrome. If you can't make it to the workshop, [the worksheet](https://docs.google.com/document/d/1UB5BVBh9uaZ1thVPeweP2kbEDAZdKWM9l_qBghEbREk/edit?usp=sharing) should still be useful. 

Reviewer: ```suggestion
This workshop will take place on [Zoom](https://asu.zoom.us/j/85894518643?pwd=S0R4MVY1SktUeWJIazYvRkRDUUtlUT09). Current committee members will start with a brief presentation on the nomination process and the responsibilities of committee members. We will then work through [a worksheet](https://docs.google.com/document/d/1UB5BVBh9uaZ1thVPeweP2kbEDAZdKWM9l_qBghEbREk/edit?usp=sharing) to help participants generate ideas and material for the nomination statement. Finally, we will give you time to draft your full statement while remaining available to answer any remaining questions and provide feedback. We especially encourage you to come to the workshop if you have questions about what constitutes a relevant experience or if you are struggling with imposter syndrome. If you can't make it to the workshop, [the worksheet](https://docs.google.com/document/d/1UB5BVBh9uaZ1thVPeweP2kbEDAZdKWM9l_qBghEbREk/edit?usp=sharing) should still be useful.
```
